src/layer/track.js

```diff
@@ -0,0 +1,147 @@
+const unpavedSelect = ["!=", ["get", "surface"], "paved"];
+const pavedSelect = ["==", ["get", "surface"], "paved"];
+const bridgeSelect = ["==", ["get", "brunnel"], "bridge"];
+const fordSelect = ["==", ["get", "brunnel"], "ford"];
+const notFordSelect = ["!=", ["get", "brunnel"], "ford"];
+const opacity = ["interpolate", ["exponential", 1.2], ["zoom"], 12, 0, 13, 1];
+const getBrunnel = ["get", "brunnel"];
+
+export const track = {
+  id: "highway-track",
+  type: "line",
+  source: "openmaptiles",
+  "source-layer": "transportation",
+  filter: ["all", trackSelect, unpavedSelect],
+  minzoom: 12,
+  paint: {
+    "line-color": ["match", getBrunnel, "ford", Color.waterLine, "#d4b791"],
+    "line-opacity": opacity,
+    "line-blur": 0.75,
+    "line-width": 0.5,
+    "line-dasharray": [12, 3],
+    "line-offset": 0,
+    "line-gap-width": [
+      "interpolate",
+      ["exponential", 1.2],
+      ["zoom"],
+      13,
+      0.7,
+      20,
+      6,
+    ],
+  },
+};
+
+export const pavedTrack = {
+  id: "highway-track-paved",
+  type: "line",
+  source: "openmaptiles",
+  "source-layer": "transportation",
+  filter: ["all", trackSelect, pavedSelect],
+  minzoom: 12,
+  paint: { ...track.paint },
+};
+pavedTrack["paint"]["line-dasharray"] = [1, 0];
+
+export const trackBridge = {
+  id: "highway-track-bridge",
+  type: "line",
+  source: "openmaptiles",
+  "source-layer": "transportation",
+  filter: ["all", trackSelect, unpavedSelect, bridgeSelect],
+  minzoom: 12,
+  paint: { ...track.paint },
+};
+
+export const pavedTrackBridge = {
+  id: "highway-track-paved-bridge",
+  type: "line",
+  source: "openmaptiles",
+  "source-layer": "transportation",
+  filter: ["all", trackSelect, pavedSelect, bridgeSelect],
+  minzoom: 12,
+  paint: { ...pavedTrack.paint },
+};
+
+// Bridge casing layers
+export const bridgeCasing = {
+  id: "track-bridge-casing",
+  type: "line",
+  source: "openmaptiles",
+  "source-layer": "transportation",
+  filter: ["all", bridgeSelect, trackSelect],
+  minzoom: 13,
+  layout: {
+    "line-cap": "butt",
+    "line-join": "bevel",
+    visibility: "visible",
+  },
+  paint: {
+    "line-color": "black",
+    "line-opacity": opacity,
+    "line-width": [
+      "interpolate",
+      ["exponential", 1.2],
+      ["zoom"],
+      13,
+      1.1,
+      20,
+      11,
+    ],
+  },
+};
+// Bridge casing layers
+export const bridgeFill = {
+  id: "track-bridge-fill",
+  type: "line",
+  source: "openmaptiles",
+  "source-layer": "transportation",
+  filter: ["all", bridgeSelect, trackSelect],
+  minzoom: 13,
+  layout: {
+    "line-cap": "butt",
+    "line-join": "bevel",
+    visibility: "visible",
+  },
+  paint: {
+    "line-color": Color.backgroundFill,
+    "line-opacity": opacity,
+    "line-width": [
+      "interpolate",
+      ["exponential", 1.2],
+      ["zoom"],
+      13,
+      1.0,
+      20,
+      10,
+    ],
+  },
+};
+
+export const legendEntries = [
+  {
+    description: "Land access track",
```

Comment:
This diplomatic wording is more appropriate in contexts like id-tagging-schema and the OSM Wiki, where it’s important for mappers to apply the right tag and not apply the wrong tag. But we have much more latitude in a legend, where the reader just has to get the gist of what distinguishes this entry from the ones above it. For comparison, the entry for expressways condenses an entire highly technical wiki article into just a few words. That’s what you’ve done here, but out of context, “land access” raises more questions than it answers.

In my opinion, even a description as simple as “Unpaved track” or “Unpaved off-road track” would suffice. I don’t think the reader would be left with the impression that they’re looking at running tracks or railroad tracks (which has its own section). Many print maps say things like “Vehicle tracks” or “4×4 tracks”, but we aren’t distinguishing by mode of transportation here.

Reply:
"track" isn't really a word we use for that here though. The closest American-ism is a "two-track road" in my vernacular.

Reply:
“Track” or “tracks” is a word in American English. It’s more general than `highway=track` (e.g., train tracks, farm tracks, animal tracks), but still close enough that the reader would not be mislead by seeing these lines described as tracks. As a preset name, it would cause mappers to dilute the tag, but dilution doesn’t happen quite like that with a map legend.

Part of the reason we’re in this quandary is that conventional transportation maps don’t even distinguish this notion of a `highway=track`. They do distinguish local roads by surface or access with captions like “unpaved local road” or “trail”, or they omit them from the map in the first place.

<img width="268" alt="Utah DOT 2007" src="https://user-images.githubusercontent.com/1231218/217153371-fed55830-6442-428d-9450-430c50db145a.png"> <img width="278" alt="Hallwag California" src="https://user-images.githubusercontent.com/1231218/217153744-06b13113-4338-484b-842c-ff8cafa55823.png"> <img width="241" alt="NMDOT 2010 compact" src="https://user-images.githubusercontent.com/1231218/217153821-a8756110-bf5b-4fe2-96b0-7abebe5585f4.png"> <img width="211" alt="Clermont County Engineer (Engels) 2004" src="https://user-images.githubusercontent.com/1231218/217154857-eae7318b-c862-4d13-9c0a-1a96dfcf2dfa.png"> <img width="310" alt="Arizona Office of Tourism 2021" src="https://user-images.githubusercontent.com/1231218/217155516-af7c11d0-9611-4d16-ad46-4038b30c5152.png">

This outdoors-oriented atlas exemplifies the kitchen-sink approach:

<img width="199" alt="Benchmark Arizona" src="https://user-images.githubusercontent.com/1231218/217155141-91141082-fc7a-456c-9d6f-d9e7876c5797.png">

> <!-- -->* Other roads in urban areas are typically paved public streets. In rural areas they include local back roads, rough 4WD routes, private roads, and logging roads. Some of these roads are usable, but others are closed to public use, or seasonally impassable, or both. Inquire locally before attempting to drive “Other Roads.”

Reply:
Thanks for the feedback. Shortened to `Unpaved track` and `Paved track` in 15a832e.